docs/examples/FontFallbackExample/src/FontFallbackExample.ls

```diff
@@ -0,0 +1,43 @@
+package
+{
+    import loom.Application;
+    import loom2d.display.Graphics;
+    import loom2d.display.Quad;
+    import loom2d.display.Shape;
+    import loom2d.display.StageScaleMode;
+    import loom2d.display.SVG;
+    import loom2d.display.TextFormat;
+
+    public class FontFallbackExample extends Application
```

Comment:
Do we need our own fallback example for this purpose? Should we reuse the NanoVG test?